### ISSUE TITLE
Issues when Generating Docking Branch

### ISSUE THREAD

**Issue by pedromsilvapt** [cimgui/cimgui#99]
I was generating the docking branch and was faced with two issues (that were fairly simple but took me quite some time to figure out, since I'm not very familiar with Lua).

The first one stems from this line in the file [imgui.h:1607](https://github.com/ocornut/imgui/blob/d057550209d794461744a91c812bf8e9e264f32f/imgui.h#L1607)
```cpp
ImVector<ImGuiTextRange>Filters;
```
(Note the lack of space after >)

Don't know if the proper fix would be to add a space when transforming the source in cimgui, or to change the original line (even though I assume it is valid C++).

The second issue was found in [generator.lua:567](https://github.com/cimgui/cimgui/blob/master/generator/generator.lua#L567) and was due to the `Pair` not being there. I fixed it with a simple `if` around those two lines, and while it worked for me, I'm not familiar enough with the code to be sure it won't have side effects.
```lua
if structs_and_enums_table["structs"]["Pair"] then
	structs_and_enums_table["structs"]["Pair"][2]["name"] = ""
	structs_and_enums_table["structs"]["Pair"][2]["type"] = structs_and_enums_table["structs"]["Pair"][2]["type"] .. "}"
end
```

**Comment by mellinoe** [comment]
FWIW, this same line appears in the master branch of imgui as well, and makes it harder to upgrade to 1.72 (released today). For some reason, I think we encountered a very similar kind of problem in the past, but I can't recall if there was a fix on the cimgui end or if the imgui source ended up having a space inserted into it.

**Comment by sonoro1234** [comment]
I will fix that and 1.72 binding on tuesday.

**Comment by sonoro1234** [comment]
Should be fixed now
